index.js

```diff
@@ -270,25 +279,48 @@ var verificationSummary = function (success) {
 // List all views in the project
 app.get('/views', function (req, res) {
     if (framework === 'adaptivejs') {
-        getAdaptiveViews(function (err, views) {
-            if (err || !views || !views.length) {
-                res.status(500).send('Error fetching view list.');
-            }
+        if (/^.?2/.test(version)) {
+            // Do version 2 stuff
+            getAdaptive2Views(function(err, views) {
+                if (err || !views || !views.length) {
+                    res.status(500).send('Error fetching view list.');
+                }
 
-            var results = views.map(function (view) {
-                var name = view.replace(/\.js/, '');
-                var viewPath = ADAPTIVEJS_VIEW_TMPL({name: name});
-                var fixturePath = ADAPTIVEJS_FIXTURE_TMPL({name: name});
+                var results = views.map(function(view) {
+                    var name = view;
+                    var viewPath = ADAPTIVE2_VIEW_TMPL({name: name});
```

Comment:
Since this is the only bit that is different between the two versions, let's just use a single code block, with a check for the template to be used, something like:

```
var viewPath = (isAdaptive2 ? ADAPTIVE2_VIEW_TMPL : ADAPTIVE_VIEW_TMPL)({ name: name });
```